tests/dialects/test_snowflake.py

```diff
@@ -998,6 +998,15 @@ def test_snowflake(self):
 
         self.validate_identity("CREATE TABLE t (id INT PRIMARY KEY AUTOINCREMENT)")
 
+        self.validate_all(
+            "SELECT FROM_HEX_BINARY('65')",
+            write={
+                "bigquery": "SELECT FROM_HEX('65')",
+                "duckdb": "SELECT FROM_HEX('65')",
+                "snowflake": "SELECT FROM_HEX_BINARY('65')",
+            },
+        )
```

Comment:
Did you test these against each engine?

Reply:
I added the results to the PR description. Let me know if that's enough

Reply:
Appreciate it 👍